qa/rpc-tests/wallet_sapling.py

```diff
@@ -144,11 +144,13 @@ def run_test(self):
 
         # Verify importing a viewing key will update the nullifiers and witnesses correctly
         extfvk0 = self.nodes[0].z_exportviewingkey(saplingAddr0)
-        self.nodes[3].z_importviewingkey(extfvk0, "yes")
-        assert_equal(self.nodes[3].z_getbalance(saplingAddr0), Decimal('10'))
+        saplingAddrInfo0 = self.nodes[3].z_importviewingkey(extfvk0, "yes")
+        assert_equal(saplingAddrInfo0["type"], "sapling")
+        assert_equal(self.nodes[3].z_getbalance(saplingAddrInfo0["address"]), Decimal('10'))
```

Comment:
Separately check that it's the expected address, here and above and below.

Reply:
Added at https://github.com/zcash/zcash/pull/4426/commits/de5b69f160e056090535697c9444a2ffdf5c2467